### PR TITLE
Task 01-02: AR add entities and resources

### PR DESCRIPTION
**Pull request by alerzep**

### CODE REVIEW

File: common/src/main/java/pl/sdacademy/simple/model/CarAttributeEntity.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package pl.sdacademy.simple.model;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+
+@Entity
+@NoArgsConstructor
```
Comment (daniel250592) [inline common/src/main/java/pl/sdacademy/simple/model/CarAttributeEntity.java:9]: annotation from lombok?

File: common/src/main/java/pl/sdacademy/simple/model/Car.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Car {
+
+    int ID;
```
Comment (daniel250592) [inline common/src/main/java/pl/sdacademy/simple/model/Car.java:16]: @ID annotation?

File: common/src/main/java/pl/sdacademy/simple/model/Car.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    String producer;
+    String model;
+    String colour;
+    int price;
+    boolean automatic;
```
Comment (softak-code) [inline common/src/main/java/pl/sdacademy/simple/model/Car.java:21]: No private.

File: common/src/main/java/pl/sdacademy/simple/model/Car.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (softak-code) [inline common/src/main/java/pl/sdacademy/simple/model/Car.java:22]: No attributes

File: common/src/main/java/pl/sdacademy/simple/model/Car.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Car {
```
Comment (softak-code) [inline common/src/main/java/pl/sdacademy/simple/model/Car.java:14]: Better CarEntity

**Review by softak-code — CHANGES_REQUESTED** [review]

**Pull request closed without merging**